### PR TITLE
[Durable Objects] Clarify method behavior + local dev

### PR DESCRIPTION
**Pull request by cacheburner**

### CODE REVIEW

File: content/durable-objects/api/hibernatable-websockets-api.md [hunk 1]
```diff
@@ -60,7 +65,9 @@ Code updates will disconnect all WebSockets. If you deploy a new version of a Wo
 
 - {{<code>}}state.getWebSockets(tag{{<param-type>}}string{{</param-type>}}{{<prop-meta>}}optional{{</prop-meta>}}){{</code>}} : {{<type>}}Array\<WebSocket>{{</type>}}
 
-  - Gets an array of accepted WebSockets matching the given tag. Disconnected WebSockets are automatically removed from the list. Calling `getWebSockets()` with no `tag` argument will return all WebSockets.
+  - Gets an array of accepted WebSockets matching the given tag. Disconnected WebSockets <sup>1</sup> are automatically removed from the list. Calling `getWebSockets()` with no `tag` argument will return all WebSockets.
+
+  - <sup>1</sup> `getWebSockets()` may still return websockets even after `ws.close()` has been called. For example, if your server side WebSocket (the Durable Object) sends a close, but does not receive one back (and has not detected a disconnect from the client), then the connection is in the CLOSING state. The client might send more messages, so the WebSocket is technically not disconnected.
```
Comment (Maddy-Cloudflare) [inline content/durable-objects/api/hibernatable-websockets-api.md]:
```suggestion
  - <sup>1</sup> `getWebSockets()` may still return websockets even after `ws.close()` has been called. For example, if your server-side WebSocket (the Durable Object) sends a close, but does not receive one back (and has not detected a disconnect from the client), then the connection is in the [`CLOSING` state](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/readyState#value). The client might send more messages, so the WebSocket is technically not disconnected.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

File: content/durable-objects/api/hibernatable-websockets-api.md (hunk 1 above)
Comment (cacheburner) [reply content/durable-objects/api/hibernatable-websockets-api.md]: `CLOSING` is from https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/readyState#value

Comment (Maddy-Cloudflare) [reply content/durable-objects/api/hibernatable-websockets-api.md]: Thanks for letting me know. 😄 

Do you have permissions to merge this PR?

Comment (cacheburner) [reply content/durable-objects/api/hibernatable-websockets-api.md]: > Do you have permissions to merge this PR?

Nope :upside_down_face:, also I updated slightly

File: content/durable-objects/api/hibernatable-websockets-api.md [hunk 2]
```diff
@@ -32,17 +32,22 @@ Code updates will disconnect all WebSockets. If you deploy a new version of a Wo
 
 {{</Aside>}}
 
+{{<Aside type="warning" header="Support for local development">}}
+
+WebSockets do not hibernate when using local development environments such as `wrangler dev` or Miniflare. Hibernatable WebSocket events such as `webSocketMessage()` will be delivered, but your Durable Object will never be evicted from memory. Hibernation will be fully supported in local development for GA, but in the meantime, prefer `wrangler dev --remote` or deploy a test/staging namespace to production when debugging state management issues.
```
Comment (Maddy-Cloudflare) [inline content/durable-objects/api/hibernatable-websockets-api.md]:
```suggestion
WebSockets do not hibernate when using local development environments such as `wrangler dev` or Miniflare. Hibernatable WebSocket events such as `webSocketMessage()` will be delivered, but your Durable Object will never be evicted from memory. Hibernation will be fully supported in local development for GA. Use `wrangler dev --remote` or deploy a test/staging Durable Object namespace to production when debugging state management issues.
```

Comment (Maddy-Cloudflare) [reply content/durable-objects/api/hibernatable-websockets-api.md]: @MellowYarker  what is GA?

**Review by Maddy-Cloudflare — APPROVED** [review]

File: content/durable-objects/api/hibernatable-websockets-api.md (hunk 2 above)
Comment (cacheburner) [reply content/durable-objects/api/hibernatable-websockets-api.md]: Generally Available, we're still in Beta

**Review by Maddy-Cloudflare — APPROVED** [review]

**Comment by Maddy-Cloudflare** [comment]
@MellowYarker  can I merge this? 😄

File: content/durable-objects/api/hibernatable-websockets-api.md [hunk 3]
```diff
@@ -32,17 +32,22 @@ Code updates will disconnect all WebSockets. If you deploy a new version of a Wo
 
 {{</Aside>}}
 
+{{<Aside type="warning" header="Support for local development">}}
+
+WebSockets do not hibernate when using local development environments such as `wrangler dev` or Miniflare. Hibernatable WebSocket events such as `webSocketMessage()` will be delivered, but your Durable Object will never be evicted from memory. Hibernation will be fully supported in local development for GA. Use `wrangler dev --remote` or deploy a test/staging Durable Object namespace to production when debugging state management issues.
```
Comment (Maddy-Cloudflare) [inline content/durable-objects/api/hibernatable-websockets-api.md]:
```suggestion
WebSockets do not hibernate when using local development environments such as `wrangler dev` or Miniflare. Hibernatable WebSocket events such as `webSocketMessage()` will be delivered, but your Durable Object will never be evicted from memory. Hibernation will be fully supported in local development for General Availability (GA). Use `wrangler dev --remote` or deploy a test/staging Durable Object namespace to production when debugging state management issues.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Comment by cacheburner** [comment]
@Maddy-Cloudflare feel free to merge and thanks for looking at this!

**Pull request merged**